compiler/compiler.go

```diff
@@ -168,11 +168,8 @@ func WriteProgramCode(pkgs []*Archive, w *SourceMapFilter) error {
 		}
 	}
 
-	if _, err := w.Write([]byte("$synthesizeMethods();\nvar $mainPkg = $packages[\"" + string(mainPkg.ImportPath) + "\"];\n$packages[\"runtime\"].$init();\n$go($mainPkg.$init, []);\n$flushConsole();\n\n}).call(this);\n")); err != nil {
-		return err
-	}
-
-	return nil
+	_, err := w.Write([]byte("$synthesizeMethods();\nvar $mainPkg = $packages[\"" + string(mainPkg.ImportPath) + "\"];\n$packages[\"runtime\"].$init();\n$go($mainPkg.$init, []);\n$flushConsole();\n\n}).call(this);\n"))
```

Comment:
I find these kind of things are best to do on your own time, in the background, and not let other code improvements be blocked on talking about it. Consider the points I made and see how you feel later on.

My main argument for why the simpler form is better is because it's equivalent, shorter, and done more commonly in the Go project (hence, more idiomatic).